pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["maturin>=1.0", "setuptools"]
```

Comment:
Do we need setuptools? I added this while debugging, but I dont believe it's needed